Resources/Prototypes/Entities/Mobs/NPCs/asteroid.yml

```diff
@@ -90,11 +90,11 @@
     - id: FoodMeatGoliath
       amount: 3
     - id: MaterialGoliathHide1
-  - type: Prying # DeltaV: let sentient goliaths pry doors
-    pryPowered: true
-    force: true
-    speedModifier: 1.5
-    useSound:
+- type: Prying # DeltaV: let sentient goliaths pry doors
```

Comment:
you trolled indent undo this

Reply:
Should be fixed now? Unless I misunderstood